projects/wave-core/src/lib/layers/symbology/symbology-raster/symbology-raster-mapping-colorizer.component.ts

```diff
@@ -74,21 +74,21 @@ export class SymbologyRasterMappingColorizerComponent implements OnChanges, OnDe
     }
 
     updateOverflowColor(event: ColorBreakpoint) {
-        if (event && !event.equals(this.symbology.overflowColor)) {
+        if (event) {
             this.symbology.overflowColor = event;
```

Comment:
The mapping value for the overflow color is `default` color.

cf. https://github.com/umr-dbs/mapping-core/blob/ad73a21074e346515d4004256310cd35d0ec26f3/src/datatypes/colorizer.cpp#L192